ArduPlane/system.cpp

```diff
@@ -230,6 +230,28 @@ bool Plane::set_mode(Mode &new_mode, const ModeReason reason)
     Mode &old_mode = *control_mode;
     const ModeReason previous_mode_reason_backup = previous_mode_reason;
 
+    // cancel inverted flight
+    auto_state.inverted_flight = false;
```

Comment:
why does this PR change anything to do with inverted flight?

Reply:
i didn't write this line, but happy to remove it.